sdcm/utils/sstable/sstable_utils.py

```diff
@@ -53,6 +51,32 @@ def get_sstables(self, from_minutes_ago: int = 0):
         self.log.debug('Got %s sstables %s', len(selected_sstables), message)
         return selected_sstables
 
+    def is_sstable_encrypted(self, sstables=None) -> list:
+        if not sstables:
+            sstables = self.get_sstables()
+        if isinstance(sstables, str):
+            sstables = [sstables]
+        sstables_encrypted_mapping = {}
+        for sstable in sstables:
+            sstables_res = self.db_node.remoter.sudo(
+                f"{self.db_node.add_install_prefix('/usr/bin/scylla')} sstable dump-scylla-metadata"
+                # NOTE: uncomment below when https://github.com/scylladb/scylla-enterprise/issues/3461 gets fixed
+                # "  --scylla-yaml-file /etc/scylla/scylla.yaml"
```

Comment:
Looks like it is required for newly created tables: https://github.com/scylladb/scylla-enterprise/issues/3461#issuecomment-1740881817
And https://github.com/scylladb/scylla-enterprise/issues/3461 comes out to be the blocker

Reply:
False. It was problem with the scylla-bench commands.